ProcessLib/ProcessVariable.cpp

```diff
@@ -56,18 +56,23 @@ ProcessVariable::ProcessVariable(
                     //! \ogs_file_param{prj__process_variables__process_variable__boundary_conditions__boundary_condition__geometry}
                     bc_config.getConfigParameter<std::string>("geometry");
 
-            GeoLib::GeoObject const* const geometry =
-                geometries.getGeoObject(geometrical_set_name, geometry_name);
-
-            if (! geometry)
-                OGS_FATAL(
-                    "No geometry with name `%s' has been found in the "
-                    "geometrical set `%s'.",
-                    geometry_name.c_str(), geometrical_set_name.c_str());
-
-            DBUG(
-                "Found geometry type \"%s\"",
-                GeoLib::convertGeoTypeToString(geometry->getGeoType()).c_str());
+            auto const full_geometry_name =
+                geometrical_set_name + "_" + geometry_name;
+            auto const mesh_it =
+                std::find_if(begin(meshes), end(meshes),
+                             [&full_geometry_name](MeshLib::Mesh* const mesh) {
+                                 return mesh == nullptr ||
```

Comment:
Why is it OK here if `mesh` is a nullptr?

Reply:
Hmmm.. that's not ok. => assertion.